### DISCUSSION TITLE
What are the other files in the configuration folder for?

### DISCUSSION THREAD

**Discussion by sorgina** [smplayer-dev/smplayer#321] [Q&A]
Hi

 As I'm not an English speaker I had my message translated by DeepL. I hope it will be clear :) . 

I use **SMPlayer 21.8.00 (revision 9887)** and to save my settings I made a copy of the **smplayer.ini** file. But in the folder I have other files and a folder :

- A file_settings folder with inside subfolders named from 0 to 9 and from a to b.

- Some files :

favorites.m3u8
hdpi.ini
player_info.ini
playlist.ini
radio.m3u8
styles.ass
tv.m3u8

I would like to know what they are for and do I have to save them too?

-----

Salut

J'utilise **SMPlayer 21.8.00 (révision 9887)** et pour sauvegarder mes réglages j'ai fait une copie du fichier **smplayer.ini**. Mais dans le dossier j'ai d'autres fichiers et un dossier :

- Un dossier **file_settings** avec dedans des sous-dossiers nommés de **0 à 9** et de **a à b**.

- Des fichiers :

favorites.m3u8
hdpi.ini
player_info.ini
playlist.ini
radio.m3u8
styles.ass
tv.m3u8

J'aimerais savoir à quoi ils servent et est-ce que je dois aussi les sauvegarder ?

**Comment by smplayer-dev** [comment]
**file_settings**: settings of the videos you played
**favorites.m3u8**: the entries in the favorites menu
**hdpi.ini**: the hdpi settings
**player_info.ini**: info about mplayer and mpv
**playlist.ini**: the playlist settings
**radio.m3u8**: the entries of the menu Open -> Radio
**styles.ass**: the style for subtitles
**tv.m3u8**: the entries of the menu Open -> TV.
